dfa/dataset.py

```diff
@@ -0,0 +1,86 @@
+from pathlib import Path
+from random import Random
+from typing import List
+
+import numpy as np
+import torch
+from torch.nn.utils.rnn import pad_sequence
+from torch.utils.data.dataloader import DataLoader
+from torch.utils.data.dataset import Dataset
+from torch.utils.data.sampler import Sampler
+
+from dfa.utils import unpickle_binary
+
+
+class AlignerDataset(Dataset):
+
+    def __init__(self, item_ids: List[str], mel_dir: Path, token_dir: Path):
+        super().__init__()
+        self.item_ids = item_ids
+        self.mel_dir = mel_dir
+        self.token_dir = token_dir
+
+    def __getitem__(self, index):
+        item_id = self.item_ids[index]
+        mel = torch.load(self.mel_dir / f'{item_id}.pt')
+        tokens = torch.load(self.token_dir / f'{item_id}.pt')
```

Comment:
numpy objects here might be easier to inspect than torch objects?

Reply:
though they would have been converted to torch tensors every batch